ui/src/app/login/controllers/LoginCtrl.js

```diff
@@ -24,6 +25,14 @@
       $state.go('signup');
     };
 
+    $scope.toggle = function toggleTheme() {
```

Comment:
This thing seems stateless. If I refresh the page theme will be back to white. Make it statefull with cache or cookies.
Also change the position of toggle button to top right corner and change the normal button to toggle button design

Reply:
@vivonk check LoginCtrl file now as this time I have used toggle-button and function made contains params for toggle-button's state :blush:

Reply:
Ya that's true @singhrohan62
But if we keep such track whether open in dark theme or light theme from beginning of theme modification then it would be great. And the same can be used in Login and other pages as well.
Else we have to again modify the code base of login for minimal change only.
Let me know whether you can implement this or not 😃.

Reply:
@vivonk I am facing some problems regarding this feature. Please reply when you read ASAP.

Reply:
@singhrohan62 yes tell. Also mention the problem or comment whenever you need help from someone with context, instead to just ping 😄😄

Reply:
@vivonk I tried to contact you via gitter but neither you read nor you replied. Nonetheless, the problem is that I am facing issues regarding enabling dark theme for both home view and login view. I think that we can put 'switch theme' option in the home view rather than in login view. What are your thoughts? and kindly reply to the message I sent you via gitter. Thanks!

Reply:
Moreover, I was just playing with the code, in the `MainController.js`, the `logout()` function, I passed a `console.log("Logout function fired!")` log statement. Surprisingly, it didn't showed up in the browser console :astonished: The code was
`var logout = function () {
      console.log("logout function fired");
      UserService.cleanUpStorage();
    };`

Reply:
@vivonk please reply when you read ASAP

Reply:
> Moreover, I was just playing with the code, in the `MainController.js`, the `logout()` function, I passed a `console.log("Logout function fired!")` log statement. Surprisingly, it didn't showed up in the browser console astonished The code was
> `var logout = function () { console.log("logout function fired"); UserService.cleanUpStorage(); };`

@vivonk did you check this on your local PC. If I am doing something wrong, kindly tell me.

Reply:
Sure @singhrohan62